FlowersStore/Scenes/FlowersList/TestPeriphery.swift

```diff
@@ -0,0 +1,32 @@
+//
+//  TestPeriphery.swift
+//  FlowersStore
+//
+//  Created by Leonir Deolindo on 02/05/22.
+//
+
+import Foundation
+import UIKit
+
+protocol Test {
```

Comment:
<!--
  0 failure: 
  1 warning:  Protocol 'Test' i...
  
  
  DangerID: danger-id-Danger;
  File: FlowersStore/Scenes/FlowersList/TestPeriphery.swift;
  Line: 11;
-->

- :warning: Protocol 'Test' is unused